regress/node_expected

```diff
@@ -1,3 +1,3 @@
 t1|SRID=10;MULTILINESTRING((0 0,5 0),(5 -5,5 0),(5 0,10 0),(5 0,5 5))
-t2|SRID=10;MULTILINESTRING((0 0,8 0),(8 0,10 0,15 0,20 0),(20 0,25 0))
+t2|SRID=10;LINESTRING(0 0,8 0,10 0,15 0,20 0,25 0)
```

Comment:
the expectance is that original nodes are retained, please don't change that (maybe it should be documented)